capstone/capweb/templates/about.html

```diff
@@ -139,7 +140,19 @@ <h2 class="subtitle">
                 </li>
               </ul>
             </div>
-
+            {# ==============> DIGITIZATION PROCESS <============== #}
+            <div class="page-section">
+              <a name="digitization"></a>
+              <h2 class="subtitle">
+                Digitization Process
+              </h2>
+              <p>
+                We created this data by digitizing roughly 40 million pages of court decisions contained in roughly 40,000 bound volumes owned by the Harvard Law School Library.
+              </p>
+              <p>
+                Members of our team created metadata for each volume, including a unique barcode, reporter name, title, jurisdiction, publication date and other volume-level information. We then used a high-speed scanner to produce JP2 and TIF images of every page. A vendor then used OCR to extract the text of every case, creating case-level XML files. Key metadata fields, like case name, citation, court and decision date, were corrected for accuracy, while the text of each case was left as raw OCR output. In addition, for cases from volumes not yet in the public domain, our vendor redacted any headnotes.
```

Comment:
I think I'd prefer "TIFF" to "TIF" here, and the first instance of "OCR" might be "optical character recognition software".